src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -310,17 +310,32 @@ private static void handleFailedConfigs(
     /**
      * Generates a report at the {@code repoReportDirectory}.
      */
-    public static void generateEmptyRepoReport(String repoReportDirectory, String displayName) {
+    public static List<Path> generateEmptyRepoReport(String repoReportDirectory, String displayName) {
         CommitReportJson emptyCommitReportJson = new CommitReportJson(displayName);
-        FileUtil.writeJsonFile(emptyCommitReportJson, getIndividualCommitsPath(repoReportDirectory));
-        FileUtil.writeJsonFile(Collections.emptyList(), getIndividualAuthorshipPath(repoReportDirectory));
+
+        List<Path> generatedFiles = new LinkedList<>();
+
+        FileUtil.writeJsonFile(emptyCommitReportJson, getIndividualCommitsPath(repoReportDirectory))
+                .ifPresent(generatedFiles::add);
+        FileUtil.writeJsonFile(Collections.emptyList(), getIndividualAuthorshipPath(repoReportDirectory))
+                .ifPresent(generatedFiles::add);
+
+        return generatedFiles;
     }
 
-    private static void generateIndividualRepoReport(
+    /**
+     * Generates a report for a single repository at {@code repoReportDirectory}.
+     */
+    private static List<Path> generateIndividualRepoReport(
```

Comment:
Same here

Reply:
Done. I've indicated the return value in the Javadoc.